resources/es-mx.yml

```diff
@@ -1,8 +1,8 @@
 name: Towny
-version: 0.50
-language: español
+version: 0.48
```

Comment:
Dont know how, but this file is being overwritten with an older version.

Reply:
Nice catch - that's probably my fault, but an easy fix.

Reply:
Should be fixed with commit 7a563df